### PR TITLE
1FA possible also for yubikey-luks-open

### PR DESCRIPTION
**Pull request by Szewcson**
Adding support for 1FA also for yubikey-luks-open

### CODE REVIEW

File: yubikey-luks-open [hunk 1]
```diff
@@ -51,12 +50,12 @@ else
...
 if [ "$HASH" = "1" ]; then
-    P1=$(printf %s "$P1" | sha256sum | awk '{print $1}')
-    if [ "$DBG" = "1" ]; then echo "Password hash: $P1"; fi
+	P1=$(printf %s "$P1" | sha256sum | awk '{print $1}')
+		if [ "$DBG" = "1" ]; then echo "Password hash: $P1"; fi
 fi
 
-R="$(printf %s "$P1" | ykchalresp -"$YUBIKEY_LUKS_SLOT" -i- 2>/dev/null || true)"
-if [ "$DBG" = "1" ]; then echo "Yubikey response: $R"; fi
+R="$(printf %s "$P1" | ykchalresp -2 -i- 2>/dev/null || true)"
```
Comment (Vincent43) [inline yubikey-luks-open]: Hardcoding slot 2 is rather poor fix that will render all keys with slot 1 unusable. Also formatting change is usually bad idea. It makes actual changes pretty unreadable and breaks commit history.

Comment (Szewcson) [reply yubikey-luks-open]: Sorry, pushed change based on wrong version of file - one I got from debian repo and not the one from github rep (I know that "It was late in the night, and I was over the beer or two" is not an excuse :P ).  Now I fix it.

File: yubikey-luks-open [hunk 2]
```diff
@@ -86,7 +90,7 @@ else
     expect_script="$expect_script$line${_n}"
     done <<EXPECTSCRIPT
         set timeout -1;
-        spawn udisksctl unlock -b "$DISK";
+        spawn udisksctl unlock --object-path "$NAME" -b "$DISK";
```
Comment (Vincent43) [inline yubikey-luks-open]: Could you specify why this change is needed? Also does it even work? The manual says:
```
-p, --object-path=OBJECT
           Specify a device by the UDisks internal object path without the /org/freedesktop/UDisks2 prefix. For example block_devices/sda for the
           /dev/sda disk.
```
So passing the `$NAME` variable looks invalid here. I confirmed it doesn't work while current `-b "$DISK"` works so I can't reason what's the goal of this.

Comment (Szewcson) [reply yubikey-luks-open]: You're right again. That what fix mapper name for me was using sudo, not that change. It looks like there is no fix for udisksctl.

**Pull request closed without merging**